server/controllers/finance/reports/debtors/annual_clients_report.js

```diff
@@ -41,6 +64,9 @@ async function setupAnnualClientsReport(options, enterprise) {
   const skipCols = 2 + shouldIncludeCashClients + (showLockedColumn ? 1 : 0);
   const numCols = 6 + skipCols;
 
+  const solventClient = (filterDebtorValue === 0);
```

Comment:
Since "filterDebtorValue" is only used to filter in/out solvent/insolvent clients, why not rename it to more readable, such as  "filterInsolventClients" or as you have in the code below "filterDebtorInsolvant".